lib/puma/client.rb

```diff
@@ -178,19 +183,13 @@ def reset(fast_check=true)
           raise HttpParserError,
             "HEADER is longer than allowed, aborting client early."
         end
-
-        return false
-      else
-        begin
-          if fast_check && @to_io.wait_readable(FAST_TRACK_KA_TIMEOUT)
```

Comment:
With this code removed, should we also remove the `FAST_TRACK_KA_TIMEOUT` assignment?